projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/forgot-password.a11y-e2e.cy.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+describe('Forgot Password Page Accessibility', () => {
```

Comment:
```suggestion
describe('Forgot Password Page Accessibility',{ testIsolation: false }, () => {
```

We can take advantage of disabling test isolation. This makes our tests run much faster.

Reply:
I've just updated.